apps/opencs/model/doc/savingstages.hpp

```diff
@@ -37,10 +37,10 @@ namespace CSMDoc
             OpenSaveStage (Document& document, SavingState& state, bool projectFile);
             ///< \param projectFile Saving the project file instead of the content file.
 
-            virtual int setup();
+            virtual int setup() override;
```

Comment:
Here's one.

As `override` implies `virtual`, the redundant `virtual` is pointlessly verbose. In general, it's best for code to only say what it needs to.

Reply:
So there is no real difference?

Reply:
Anyway, I managed to make clang-tidy to work properly, so there is even more changes now.